### ISSUE TITLE
Websocket connection failing

### ISSUE THREAD

**Issue by kukirokuk** [ethand91/mediasoup3-record-demo#60]
I try to open localhost:8080 and there is websocket connection fail in chrome console:
`index.js:12 WebSocket connection to 'wss://localhost:3000/' failed`
I have changed websocket port, allowed unsecure connection in chrome site settings but nothing works.
May be this is relevant, I have the same error when trying open official mediasoud demo page https://v3demo.mediasoup.org

**Comment by thedeephakani** [comment]
facing same issue.

**Comment by ethand91** [comment]
Are you running a Firewall etc.?

**Comment by thedeephakani** [comment]
No, I am working on localhost and have no firewall restriction.

**Comment by abhinavsy** [comment]
Give your own ip  address or give a public ip adress  in announced ip at config.js file.This solved the problem for me

**Comment by asubramania** [comment]
Please fix the "cors" on the server side:

//const wss = new WebSocket.Server({ server: httpsServer });
const wss = new WebSocket.Server({
  server: httpsServer,
  verifyClient: (info) => {
    const origin = info.origin;
    // Allow all origins for now, for debugging purposes:
    return true;
    // For production, you should check the origin, as shown previously.
  }
});

Hope this helps!

**Comment by anasmohammed361** [comment]
same issue with this, i've also added cors but the error seem to persist

**Comment by anasmohammed361** [comment]
I switched my ws server to http and altered the url in the clien t, now it seems to work fine

**Comment by sorokinvj** [comment]
both Chrome and Safari was not happy with certificate, so refactoring it to use http instead of https helped
